tensorflow_datasets/audio/freesound_audio_tagging_2019.py

```diff
@@ -0,0 +1,152 @@
+from pathlib import Path
+
+import kaggle
```

Comment:
TFDS also have a kaggle downloader on `core/download/kaggle.py`. Please check it.